### DISCUSSION TITLE
[vue-composition-api] must call Vue.use(VueCompositionAPI) before using any function.

### DISCUSSION THREAD

**Discussion by antlionguard** [vuejs/pinia#1209] [Help and Questions] [answered]
### Reproduction

https://github.com/Luferov/bea

### Steps to reproduce the bug

- yarn install
- yarn dev

### Expected behavior

runs without error.

### Actual behavior

"[vue-composition-api] must call Vue.use(VueCompositionAPI) before using any function" error occurs all pinia used pages.
The error related with : https://github.com/nuxt/framework/issues/3904. This issue still open but i think the error related with pinia.

**Comment by AndreyYolkin** [comment]
Can confirm this, used [@nuxt-alt/pinia](https://github.com/Teranode/nuxt-module-alternatives/tree/master/%40nuxtjs-alt/pinia) and got the same result in production

**Reply by antlionguard** [reply]
<img width="559" alt="image" src="https://user-images.githubusercontent.com/26603638/162194368-f476c9af-2e75-48bc-96bd-c8fa0b5e0f21.png">

I'm using bridge. And i got this error.

**Reply by AndreyYolkin** [reply]
Thats because author described this in package, but there are no nuxt3-only related code, all built using @nuxt/kit, I just removed declaration from package

**Reply by antlionguard** [reply]
i removed compability version code and i tried but i got same error:
 ``` [vue-composition-api] must call Vue.use(VueCompositionAPI) before using any function. ```

**Reply by AndreyYolkin** [reply]
Yes, that's what i said, different module leads to the same result ._.

**Reply by antlionguard** [reply]
So what is the solution? What should i do :D  Because i cannot use newest version of nuxt-bridge. My development process stuck :(

**Reply by Luferov** [reply]
Cool! It’s look like - you shouldn’t use pinia in nuxt2 and nuxt-bridge:)
But I want to use pinia)

**Comment by antlionguard** [comment]
@posva there's any idea for this error? that error blocking our development process completely :(

**Comment by antlionguard** [comment]
Extra info: 

The error appears while calling ```vueDemi.effectScope(true)``` function.

<img width="629" alt="image" src="https://user-images.githubusercontent.com/26603638/165483912-34ae09da-31c7-481d-821d-096aff64c26d.png">

**Reply by Luferov** [reply]
Very cool. Thank you for your work. I want to dig into this assembly on my weekends, because the work has become. And I can't upgrade to nuxt3.

**Reply by antlionguard** [reply]
Another info. This error appears after [3.0.0-27470185.38e9259](https://www.npmjs.com/package/@nuxt/bridge-edge/v/3.0.0-27470185.38e9259) version.

**Comment by antlionguard** [comment, marked as answer]
This error resolved in new versions of nuxt bridge.

**Comment by edwh** [comment]
I'm not using Bridge, just pure Nuxt3.

**Reply by antlionguard** [reply]
This discussion opened for bridge version of nuxt. You should open another discussion or issue if you have errors with nuxt3.
